docs/standard/base-types/string-comparison-net-5-plus.md

```diff
@@ -49,7 +49,7 @@ These specific rules aren't enabled by default. To enable them and show any viol
 
 ```xml
 <PropertyGroup>
-  <AnalysisMode>AllEnabledByDefault</AnalysisMode>
+  <AnalysisMode>All</AnalysisMode>
```

Comment:
Is this accurate? The file name indicates .NET 5+ but the value used here is .NET 6.

Reply:
I guess I assume that most people using .NET 5 will upgrade to .NET 6 since it's LTS. I could add a note block to use a different value for .NET 5 if you think it's necessary.